benches/banking_stage.rs

```diff
@@ -97,110 +37,120 @@ fn check_txs(receiver: &Receiver<Signal>, ref_tx_count: usize) {
 
 #[bench]
 fn bench_banking_stage_multi_accounts(bencher: &mut Bencher) {
-    let tx = 10_000_usize;
+    let txes = 1024 * NUM_THREADS;
     let mint_total = 1_000_000_000_000;
     let mint = Mint::new(mint_total);
-    let num_dst_accounts = 8 * 1024;
-    let num_src_accounts = 8 * 1024;
-
-    let srckeys: Vec<_> = (0..num_src_accounts).map(|_| Keypair::new()).collect();
-    let dstkeys: Vec<_> = (0..num_dst_accounts)
-        .map(|_| Keypair::new().pubkey())
-        .collect();
-
-    let transactions: Vec<_> = (0..tx)
-        .map(|i| {
-            Transaction::new(
-                &srckeys[i % num_src_accounts],
-                dstkeys[i % num_dst_accounts],
-                i as i64,
-                mint.last_id(),
-            )
-        }).collect();
 
     let (verified_sender, verified_receiver) = channel();
-    let (signal_sender, signal_receiver) = channel();
     let packet_recycler = PacketRecycler::default();
-
-    let setup_transactions: Vec<_> = (0..num_src_accounts)
-        .map(|i| {
-            Transaction::new(
-                &mint.keypair(),
-                srckeys[i].pubkey(),
-                mint_total / num_src_accounts as i64,
-                mint.last_id(),
-            )
+    let bank = Arc::new(Bank::new(&mint));
+    let dummy = Transaction::new(&mint.keypair(), mint.keypair().pubkey(), 1, bank.last_id());
+    let transactions: Vec<_> = (0..txes)
+        .into_par_iter()
+        .map(|_| {
+            let mut new = dummy.clone();
+            let from: Vec<u8> = (0..64).map(|_| thread_rng().gen()).collect();
+            let to: Vec<u8> = (0..64).map(|_| thread_rng().gen()).collect();
+            let sig: Vec<u8> = (0..64).map(|_| thread_rng().gen()).collect();
+            new.keys[0] = Pubkey::new(&from[0..32]);
+            new.keys[1] = Pubkey::new(&to[0..32]);
+            new.signature = Signature::new(&sig[0..64]);
+            new
         }).collect();
-
+    transactions.iter().for_each(|tx| {
+        let fund = Transaction::new(
+            &mint.keypair(),
```

Comment:
This always transfers from the mint right? This is meant to transfer from multiple source accounts.